perfmetrics/scripts/vm_metrics/vm_metrics.py

```diff
@@ -295,8 +313,7 @@ def main() -> None:
   test_type = sys.argv[5]
   worksheet_name = sys.argv[6]
   vm_metrics = VmMetrics()
-  vm_metrics.fetch_metrics_and_write_to_google_sheet(start_time_sec, end_time_sec, instance,
-                                                     period, test_type, worksheet_name)
+  vm_metrics.fetch_metrics_and_write_to_google_sheet(start_time_sec, end_time_sec, instance, period, test_type, worksheet_name)
```

Comment:
Can you preserve the previous formatting here and everywhere (<80 chars per line)

Reply:
done